src/ca_biositing/pipeline/ca_biositing/pipeline/utils/reviewed_api_mappings.py

```
@@ -30,20 +57,1356 @@
+    '34 MAJOR VEGETABLES (VARIES BY SEASON)': 'not in QuickStats API list',
+    '9 MAJOR VEGETABLES (VARIES BY SEASON)': 'not in QuickStats API list',
+    'ALFALFA & ALFALFA MIXTURES - NEW SEEDINGS': 'not in QuickStats API list',  # heuristic unchanged — find the correct QuickStats name
+    'ALL': 'not in QuickStats API list',  # heuristic unchanged — find the correct QuickStats name
+    'ALL COWS': 'not in QuickStats API list',  # heuristic guess: 'COWS' — verify before moving to OFFICIAL
+    'ALL FARMS': 'not in QuickStats API list',  # heuristic guess: 'FARMS' — verify before moving to OFFICIAL
+    'ALL NONCITRUS': 'not in QuickStats API list',  # heuristic guess: 'NONCITRUS' — verify before moving to OFFICIAL
+    'ALL NUTS': 'not in QuickStats API list',  # heuristic guess: 'NUTS' — verify before moving to OFFICIAL
+    'ANGORA': 'not in QuickStats API list',  # heuristic unchanged — find the correct QuickStats name
+    'ARTICHOKE': 'not in QuickStats API list',  # heuristic unchanged — find the correct QuickStats name
+    'BARROWS & GILTS': 'not in QuickStats API list',  # heuristic unchanged — find the correct QuickStats name
+    'BARROWS & GILTS': 'not in QuickStats API list',  # heuristic unchanged — find the correct QuickStats name
```

Ruff (0.15.5)

[error] Dictionary key literal `'BARROWS & GILTS'` repeated

Remove repeated key literal `'BARROWS & GILTS'`

(F601)